src/omnisharp/typeConversion.ts

```diff
@@ -62,7 +62,28 @@ export function toVSCodePosition(point: protocol.V2.Point): vscode.Position {
     return new vscode.Position(point.Line, point.Column);
 }
 
-export function createRequest<T extends protocol.Request>(document: vscode.TextDocument, where: vscode.Position, includeBuffer: boolean = false): T {
+export function toVSCodeTextEdit(textChange: protocol.LinePositionSpanTextChange): vscode.TextEdit {
+    return new vscode.TextEdit(toVSCodeRange(textChange), textChange.NewText);
+
+    function toVSCodeRange(textChange: protocol.LinePositionSpanTextChange): vscode.Range {
```

Comment:
Consider just using `toRange2`.

Reply:
not sure how I could have missed such a well named helper. Will update.